packages/mdc-button/mdc-button.scss

```diff
@@ -17,62 +17,34 @@
 @import "@material/elevation/mixins";
 @import "@material/ripple/mixins";
 @import "@material/theme/mixins";
+@import "@material/theme/variables";
 @import "@material/typography/mixins";
+@import "./variables";
 
 // postcss-bem-linter: define button
-
 .mdc-button {
   @include mdc-ripple-base;
-  @include mdc-ripple-bg((pseudo: "::before"));
-  @include mdc-ripple-fg((pseudo: "::after"));
-  @include mdc-theme-prop(color, text-primary-on-light);
-  @include mdc-typography-base;
+  @include mdc-ripple-bg((pseudo: "::before", base-color: $mdc-button-baseline-color-on-light, opacity: .16));
+  @include mdc-ripple-fg((pseudo: "::after", base-color: $mdc-button-baseline-color-on-light, opacity: .16));
+  @include mdc-typography(button);
 
   display: inline-block;
   position: relative;
-  min-width: 64px;
+  min-width: 88px;
   height: 36px;
-  padding: 0 16px;
+  padding: 0 8px;
```

Comment:
Also the `raised` button still have a padding of `16px` (line 98), so `mdc-button--compact` will still make a style difference there.